src/core/datatypes/phylogenetics/characterhistory/CharacterEvent.cpp

```diff
@@ -5,23 +5,20 @@
 using namespace RevBayesCore;
 
 
-CharacterEvent::CharacterEvent(void)
+CharacterEvent::CharacterEvent(void) :
+    site_index(0),
```

Comment:
Can you move this to the header, like `int site_index = 0;`?
And the same with age, event_type, and missing.